src/Components/Steps/ZipcodeStep.tsx

```diff
@@ -58,10 +52,8 @@ export const ZipcodeStep = ({ currentStepId }: ZipcodeStepProps) => {
   });
 
   const currentZipcodeValue = watch('zipcode');
-  const shouldShowCountyInput =
-    numberMustBeFiveDigitsLongRegex.test(currentZipcodeValue) &&
-    Object.keys(countiesByZipcode).includes(currentZipcodeValue);
```

Comment:
Do we need to include this in the zip code schema?

Reply:
It's already in the zipcode schema.

Reply:
It looks like it only check that the zip code is the correct length, but does not check if it is in `countiesByZipcode`.

Reply:
It does with the `refine` method.